Hakawai/Mentions/HKWMentionsPlugin.m

```diff
@@ -2013,9 +2055,9 @@ - (void)createMention:(HKWMentionsAttribute *)mention startingLocation:(NSUInteg
     NSRange rangeToTransform;
     if (HKWTextView.enableSimpleRefactor) {
         // Find where previous control character was, and replace mention at that point
-        NSRange rangeOfControlChar = [parentTextView.text rangeOfString:@"@" options:NSBackwardsSearch];
-        NSUInteger lengthOfMention = currentLocation + wordAfterCurrentLocation.length - rangeOfControlChar.location;
-        rangeToTransform = NSMakeRange(rangeOfControlChar.location, lengthOfMention);
+        NSUInteger controlCharLocation = [HKWMentionsPlugin mostRecentControlCharacterLocation:parentTextView.text controlCharacterSet:self.controlCharacterSet];
+        NSUInteger lengthOfMention = currentLocation + wordAfterCurrentLocation.length - controlCharLocation;
```

Comment:
i'm not sure if i understand this logic

Reply:
it's existing. but added a comment to explain.